src/loaders/svg/tvgSvgSceneBuilder.cpp

```diff
@@ -788,3 +788,16 @@ unique_ptr<Scene> svgSceneBuild(SvgNode* node, float vx, float vy, float vw, flo
 
     return root;
 }
+
+void applyBBoxFlag(const Scene* scene, float& vx, float& vy, float& vw, float& vh, float& w, float& h, BBoxFlag bboxFlag)
+{
+    if ((bboxFlag & BBoxFlag::Viewbox) != BBoxFlag::Viewbox) {
```

Comment:
nit: BBoxFlag::Viewbox is full masking, so the code below is same.
`if (bboxFlag != BBoxFlag::Viewbox)`

Reply:
it was my bug, doesn;t exist after changes